amplify/backend/function/risestandstrongcb34053a/src/routes/announcements.js

```diff
@@ -129,8 +131,19 @@ router.post('/', async (req, res) => {
  *         "404":
  *            description: Announcement with the specified announcementId does not exist
  */
+
 router.put('/:announcementId', async (req, res) => {
-    res.end();
+    const data = req.body;
+    if (!AWS_Auth.Auth.currentAuthenticatedUser()){
+        res.status(403)
+    }
+     else{
+        const recieved = await putAnnouncement(data);
+        if (typeof recieved === undefined){
+            res.status(404);
+        }
+        res.status(200).json(data);
```

Comment:
res.json(data) will work just fine because if you don't specify a status code it assumes 200.